eav/admin.py

```diff
@@ -19,11 +19,12 @@ def render_change_form(self, request, context, *args, **kwargs):
         view and substitute some data.
         """
         form = context['adminform'].form
+        media = context["media"]
 
         # Infer correct data from the form.
         fieldsets = self.fieldsets or [(None, {'fields': form.fields.keys()})]
         adminform = admin.helpers.AdminForm(form, fieldsets, self.prepopulated_fields)
-        media = mark_safe(self.media + adminform.media)
+        media = mark_safe(media + adminform.media)
```

Comment:
I am using Django 3.2

When I created `class ProductAdmin(BaseEntityAdmin)` the `Add another` button for inline fields disappeared.

This happens because the `self.media` returns [this](https://github.com/django/django/blob/stable/3.2.x/django/contrib/admin/options.py#L639-L651) and there is no `admin/js/inlines.js` there by default.
The `Add another` button for inlines is added on the fly by [admin/js/inlines.js](https://github.com/django/django/blob/stable/3.2.x/django/contrib/admin/static/admin/js/inlines.js) script.
The `context["media"]` contains current/nessasary [form assets](https://docs.djangoproject.com/en/3.2/topics/forms/media/) to create the button.